frontend/src/scenes/retention/RetentionTable.js

```diff
@@ -59,6 +70,32 @@ export function RetentionTable({ logic }) {
     }
     return (
         <>
+            <Row>
+                <div>
+                    <span style={{ marginRight: 5 }}>Start Day:</span>
+                    <DatePicker
+                        className="mb-2"
+                        value={selectedDate}
+                        onChange={dateChanged}
+                        allowClear={false}
+                    ></DatePicker>
+                </div>
+                <div>
+                    <span style={{ marginLeft: 10, marginRight: 5 }}>Period:</span>
+                    <Select
+                        value={dateOptions[period]}
+                        onChange={(value) => setPeriod(value)}
```

Comment:
This can just be `onChange={setPeriod}` (unless there's a linter rule that disallows it)